client/src/AddForm.js

```diff
@@ -1,97 +0,0 @@
-import React, { Component } from "react";
-import "./App.css";
-import axios from "axios";
-
-const BASE_URL = "http://localhost:3005";
-
-class AddForm extends Component {
-  constructor() {
-    super();
-    this.state = {
-      name: "",
-      image_url: "",
-      scary: {
-        level: 0
-      }
-    };
-  }
-  handleScaryLevel = event => {
-    const { scary } = this.state.scary;
-    const newScary = Object.assign({}, scary);
```

Comment:
you can also use the spread operator:
`const newScary = {...scary}`

Reply:
Thanks for the tip Luke.. I am still not very comfortable with the spread operator -- will keep experimenting until I am..